statics/js/components/topology.js

```diff
@@ -6,224 +6,226 @@ var TopologyComponent = {
 
   mixins: [apiMixin, notificationMixin],
 
-  template: '\
-    <div class="topology">\
-      <div class="col-sm-7 fill content">\
-        <div class="topology-d3">\
-          <div class="topology-legend">\
-            <strong>Topology view</strong></br>\
-            <span v-if="currTopologyFilter">{{currTopologyFilter}}</span>\
```

Comment:
just convert the 3 following "span" to "p"

Reply:
ok